modules/dxp/apps/osb/osb-faro/.gitrepo

```diff
@@ -4,8 +4,8 @@
 [subrepo]
 	autopull = false
 	cmdver = liferay
-	commit = 816318956b24a772ba367f57e93e21f268649841
+	commit = 8f2d9c2f85cfe4c253114fe19123a78e6491d814
```

Comment:
Where does this change come from?

Reply:
It's Strange... I think I didn't do it. If you go up in the conversation you will se that Continous integration from Hungary force pushed a commit. I don't know if maybe it has something to do with it